File: WeekendTask/javascript/GcdNumber.js
const prompt=require('prompt-sync')();
function greatestCommonFactor(firstNumber,secondNumber){
	let greatestNumber=0;
		for(let count=1; count<=firstNumber; count++){
		 
			if (firstNumber %count ==0 && secondNumber % count==0){
				greatestNumber = count;
				}
	
	
		}
		return greatestNumber;
		
			
	}
let firstNumber=prompt("Enter First Number ")

let secondNumber= prompt("Enter Second Number ");



console.log(greatestCommonFactor(firstNumber,secondNumber));

File: WeekendTask/python/gcd_Number.py
def greatest_common_factor(first_number, second_number):
    greatest_number = 0
    for count in range(1, first_number, second_number):
        if first_number % count == 0 and second_number % count == 0:
            greatest_number = count
            return greatest_number


first_number = int(input("Enter First Number "))
second_number = int(input("Enter Second Number "))

print(greatest_common_factor(first_number, second_number))
